RELEASE.md

```diff
@@ -242,7 +242,7 @@ Hadi Hashemi, sgt101, Shitian Ni, Shuolongbj, Siim PõDer, Simon Perkins, sj6077
 Spotlight0xff, Steffen Eberbach, Stephen Fox, superryanguo, Sven Mayer, Tapan Prakash,
 Tiago Morais Morgado, Till Hoffmann, Tj Rana, Vadim Markovtsev, vhasanov, Wei Wu,
 windead, Yan (Asta) Li, Yan Chen, Yann Henon, Yi Wang, Yong Tang, yorkie, Yuan (Terry)
-Tang, Yuxin Wu, zhengjiajin, zhongzyd, 黄璞
+Tang, Yuxin Wu, zhengjiajin, zhongzyd, 黄璞, Yasunaka Cho
```

Comment:
This list is sorted by UTF-8 order. So you should move your name to the appropriate place in the Y cohort.